extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -171,6 +173,89 @@ class AmpAccordion extends AMP.BaseElement {
     this.currentState_[contentId] = !isSectionClosedAfterClick;
     this.setSessionState_();
   }
+
+  /**
+   * Handles accordion headers clicks to expand/collapse its content.
+   * @param {!Event} event Click event.
+   * @private
+   */
+  clickHandler_(event) {
+    // Need to support clicks on any children of the header.
+    let header;
+    for (let i = 0; i < this.headers_.length; i++) {
+      const curr = this.headers_[i];
+      if (curr.contains(event.target)) {
+        header = curr;
+        break;
+      }
+    }
+    if (header) {
+      header = dev().assertElement(header);
+      event.preventDefault();
+      this.onHeaderPicked_(header);
+    }
+  }
+
+  /**
+   * Handles accordion key presses ot expand/collapse its content.
+   * @param {!Event} event keydown event.
+   */
+  keyDownHandler_(event) {
+    if (event.defaultPrevented) {
+      return;
+    }
+    const keyCode = event.keyCode;
+    switch (keyCode) {
+      case KeyCodes.UP_ARROW: /* fallthrough */
+      case KeyCodes.DOWN_ARROW:
+        this.navigationKeyDownHandler_(event);
```

Comment:
I share your concern about unnecessarily processing keyboard events that we don't care about. I think we can find a way to install the handler on the headers rather than on the whole accordion and still accommodate keyboard navigation. PTAL.